### PR TITLE
Add auth endpoint parameter filtering config

### PR DESCRIPTION
**Pull request by ThaminduR**
This PR adds authentication endpoint parameter filtering configs to deployment.toml file.

- This PR should be merged after https://github.com/wso2/carbon-identity-framework/pull/4228
- This PR should be merged along with https://github.com/wso2/identity-apps/pull/3400

Related Issue: https://github.com/wso2/product-is/issues/14878

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/identity_new_resource.toml [hunk 1]
```diff
@@ -30,4 +30,4 @@ password = "wso2carbon"
 [[resource.access_control]]
 context = "(.*)/sample-auth/(.*)"
 secure = false
-http_method = "all"
+http_method = "all"
```
Comment (sahandilshan) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/identity_new_resource.toml:33]: Add a new line at the bottom of the file

**Review by senthalan — APPROVED** [review]

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/auth/RiskBasedLoginTestCase.java [hunk 2]
```diff
@@ -301,6 +303,11 @@ public void testAuthenticationForRisk() throws Exception {
         Header locationHeader = response.getFirstHeader(OAuth2Constant.HTTP_RESPONSE_HEADER_LOCATION);
 
         String callbackUrl = DataExtractUtil.getParamFromURIString(locationHeader.getValue(), "callbackUrl");
+        if (callbackUrl==null) {
```
Comment (lashinijay) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/auth/RiskBasedLoginTestCase.java:306]: formatting issue

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/util/Utils.java [hunk 3]
```diff
@@ -254,6 +267,20 @@ public static HttpResponse sendPOSTChallengeQuestionResponse(HttpResponse respon
         return httpClient.execute(post);
     }
 
+    private static List<String> extractSecurityQuestions(HttpResponse response) throws IOException {
```
Comment (lashinijay) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/util/Utils.java:270]: Let's add a new line after method signature.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/util/Utils.java [hunk 4]
```diff
@@ -299,12 +326,37 @@ public static HttpResponse sendPOSTConsentMessage(HttpResponse response, String
...
+        }
+
         urlParameters.add(new BasicNameValuePair("sessionDataKey", sessionKey));
         urlParameters.add(new BasicNameValuePair("consent", "approve"));
         post.setEntity(new UrlEncodedFormEntity(urlParameters));
         return httpClient.execute(post);
     }
 
+    private static List<String> extractClaims(HttpResponse response) throws IOException {
+        BufferedReader rd = new BufferedReader(new InputStreamReader(response.getEntity().getContent()));
```
Comment (lashinijay) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/util/Utils.java:351]: Formatting issue, add a new line after the method signature.

**Pull request merged**